packages/web/src/components/common/header/Header.styles.ts

```diff
@@ -4,39 +4,39 @@ import mixins from "@styles/mixins";
 import { Z_INDEX } from "@styles/zIndex";
 
 export const HeaderWrapper = styled.header`
-  position: fixed;
+  ${mixins.flexbox("column", "center", "center")};
   top: 0;
+  position: fixed;
   z-index: ${Z_INDEX.fixed};
+  max-width: 1920px;
```

Comment:
If the browser size is large, this attribute will cause the header to be skewed to the left. 
Is there a reason for this attribute?

Reply:
It seems it was a mistake. Deleted. thank you !